mmdet/models/backbones/darknet.py

```diff
@@ -197,3 +211,68 @@ def make_conv_res_block(in_channels,
             model.add_module('res{}'.format(idx),
                              ResBlock(out_channels, **cfg))
         return model
+
+
+class Csp_conv_res_block(nn.Module):
```

Comment:
Could this function be merged in `self.make_conv_res_block()`?

Reply:
I don't think so because this v4 conv_res_block has shortcut (therefore requiring a customized `forward` function) while the original `self.make_conv_res_block()` does not. However, the original `self.make_conv_res_block()` could probably be merged into this new class if necessary.